db/db_impl/db_impl.cc

```diff
@@ -1496,6 +1496,28 @@ bool DBImpl::SetPreserveDeletesSequenceNumber(SequenceNumber seqnum) {
   }
 }
 
+Status DBImpl::GetFullHistoryTsLow(ColumnFamilyHandle* column_family,
+                                   std::string* ts_low) {
+  if (ts_low == nullptr) {
+    return Status::InvalidArgument("Input ts_low is nullptr");
```

Comment:
Nit: some confusion here. `ts_low` pointer is an input, but we generally refer to such arguments as output. Simply removing `Input` will be fine.

Reply:
fixed